### PR TITLE
Feature/df 940 jwt config

### PR DESCRIPTION
**Pull request by fabiangotzen**

### CODE REVIEW

File: web/build.gradle [hunk 1]
```diff
@@ -31,7 +31,8 @@ dependencies {
     compile "org.springframework.cloud:spring-cloud-starter-netflix-eureka-client"
     compile "org.springframework.cloud:spring-cloud-starter-config"
     compile 'ch.admin.seco.alv:alv-feign-starter'
-
+    compile "ch.admin.seco.alv:alv-jwt-starter"
+    
     compile "org.springframework.security:spring-security-web"
 
     compile "io.jsonwebtoken:jjwt"
```
Comment (birom) [inline web/build.gradle]: Could you remove this. We do not need that anymore.

Comment (fabiangotzen) [reply web/build.gradle]: removed

File: web/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/web/config/MicroserviceSecurityConfig.java [hunk 2]
```diff
@@ -23,30 +24,24 @@
...
+import org.zalando.problem.spring.web.advice.security.SecurityProblemSupport;
 
 @Configuration
 @Import(SecurityProblemSupport.class)
 public class MicroserviceSecurityConfig {
 
     @Configuration
     @Order(SecurityProperties.BASIC_AUTH_ORDER)
-    @EnableConfigurationProperties(JWTSecurityProperties.class)
     @Profile("!" + ProfileRegistry.OAUTH2)
```
Comment (birom) [inline web/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/web/config/MicroserviceSecurityConfig.java]: I think we do not use the ``oauth2`` profile any more. If this is correct we can get rid of the inner classes and move the ``configure`` methods to the outer class.

File: gradle.properties [hunk 3]
```diff
@@ -6,7 +6,7 @@ hibernate_version=5.3.7.Final
 mapstruct_version=1.2.0.Final
 liquibase_hibernate5_version=3.6
 springfox_swagger2_version=2.9.2
-alv_shared_library_version=0.0.1-build-103
+alv_shared_library_version=0.0.1-build-116
```
Comment (birom) [inline gradle.properties]: Use the latest version as discussed.

Comment (fabiangotzen) [reply gradle.properties]: done

File: web/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/web/config/MicroserviceSecurityConfig.java (hunk 2 above)
Comment (fabiangotzen) [reply web/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/web/config/MicroserviceSecurityConfig.java]: removed the inner classes

**Pull request merged**